src/dashboard/apps/consent/views.py

```diff
@@ -270,3 +270,39 @@ def _send_email(self) -> None:
             # fail silently and send a sentry log
             sentry_sdk.capture_exception(e)
             return
+
+
+class ValidatedConsentView(BreadcrumbContextMixin, TemplateView):
```

Comment:
What about using a `ListView` instead to that it can handle pagination?

https://ccbv.co.uk/projects/Django/5.0/django.views.generic.list/ListView/